Corona-Warn-App/src/test/java/de/rki/coronawarnapp/presencetracing/checkins/checkout/ContactJournalCheckInEntryCreatorTest.kt

```diff
@@ -0,0 +1,87 @@
+package de.rki.coronawarnapp.presencetracing.checkins.checkout
+
+import de.rki.coronawarnapp.contactdiary.model.DefaultContactDiaryLocation
+import de.rki.coronawarnapp.contactdiary.storage.repo.ContactDiaryRepository
+import de.rki.coronawarnapp.eventregistration.checkins.CheckIn
+import io.mockk.MockKAnnotations
+import io.mockk.coEvery
+import io.mockk.coVerify
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.just
+import io.mockk.runs
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.test.runBlockingTest
+import okio.ByteString.Companion.encode
+import org.joda.time.Instant
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import testhelpers.BaseTest
+
+class ContactJournalCheckInEntryCreatorTest : BaseTest() {
+
+    @MockK lateinit var contactDiaryRepo: ContactDiaryRepository
+
+    private val testCheckIn = CheckIn(
+        id = 42L,
+        traceLocationId = "traceLocationId1".encode(),
```

Comment:
should it be `encodeBase64` ?

Reply:
yes. but as I wrote advanced testing is coming and variable is not relevant yet